pkgs/top-level/perl-packages.nix

```diff
@@ -746,9 +783,9 @@ let
          --replace 'sed' '${pkgs.gnused}/bin/sed'
     '';
     meta = {
-      description = "A container for functions of the ClusterSSH programs";
+      description = "Cluster administration tool";
       license = with lib.licenses; [ artistic1 gpl1Plus ];
-      homepage = "https://github.com/duncs/clusterssh/wiki";
+      homepage = "http://github.com/duncs/clusterssh/wiki";
```

Comment:
I am not sure how it comply with nixpkg's `http`->`https` rewriting <s>paranoia</s> initiative, but it is a literal string from CPAN.
I do not even check if the CPAN's homepage string contains an URL